tests/integration/kubernetes/tests_common.sh

```diff
@@ -111,7 +111,7 @@ exec_host() {
 	# [bats-exec-test:38] INFO: k8s configured to use runtimeclass
 	# bash: line 1: $'\r': command not found
 	# ```
-	output="$(kubectl debug -qi "node/${node}" --image=ghcr.io/linuxcontainers/alpine:latest -- chroot /host bash -c "${command}" | tr -d '\r')"
+	output="$(kubectl debug -qi "node/${node}" --image=quay.io/libpod/ubuntu:latest -- chroot /host bash -c "${command}" | tr -d '\r')"
```

Comment:
Let's use this image instead since it looks like the `libpod` one hasn't been updated in years.
```suggestion
	output="$(kubectl debug -qi "node/${node}" --image=quay.io/bedrock/ubuntu:latest -- chroot /host bash -c "${command}" | tr -d '\r')"
```